openhtf/exe/test_state.py

```diff
@@ -46,27 +46,16 @@ class TestState(object):
   """ This class handles tracking the state of the test.
 
   Args:
-    config: The config being used for this test.
     test: openhtf.Test instance describing the test to run.
+    plug_map: dict mapping plug name to instances.
+    dut_id: DUT identifier, if it's known, otherwise None.
   """
-  State = Enum('State', ['RUNNING', 'ERROR', 'TIMEOUT', 'ABORTED',
-                         'WAITING', 'FAIL', 'PASS', 'CREATED'])
-
-  _PHASE_RESULT_TO_STATE = {
-      phase_data.PhaseResults.CONTINUE: State.WAITING,
-      phase_data.PhaseResults.REPEAT: State.WAITING,
-      phase_data.PhaseResults.FAIL: State.FAIL,
-      phase_data.PhaseResults.TIMEOUT: State.TIMEOUT,
-  }
+  State = Enum('State', ['CREATED', 'RUNNING', 'COMPLETED'])
```

Comment:
yeah we can add that back in when we plumb that through - ABORTED was unused here and I'd rather keep this as minimal as possible and only add things in when they are supported.